packages/instructor-stream/src/utils/tokenizer.ts

```
@@ -152,490 +150,487 @@
+
+          i--
+          this.state = TokenizerStates.START
+          this.emitNumber()
+          continue
+        // @ts-expect error fall through case
+        case TokenizerStates.NUMBER_AFTER_E:
+          if (n === charset.PLUS_SIGN || n === charset.HYPHEN_MINUS) {
+            this.bufferedNumber.appendChar(n)
+            this.state = TokenizerStates.NUMBER_AFTER_E_AND_SIGN
+            continue
+          }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)